pkg/machine/qemu/machine.go

```diff
@@ -572,14 +580,14 @@ func (v *MachineVM) Start(name string, _ machine.StartOptions) error {
 		time.Sleep(wait)
 		wait++
 	}
+	defer conn.Close()
```

Comment:
This must be done after the error check below, otherwise it causes a panic on error